.github/workflows/verify.yaml

```diff
@@ -35,7 +35,10 @@ on:
     paths-ignore:
       - 'charts/**'
     branches:
-      - '*'
+      - releases
```

Comment:
any particular reason for this? seems to not pertain to this PR

Reply:
i guess we could just remove the `branches` section as it works as a filter

Reply:
Done